### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by derekcaneja**
Update react and react dom to 15.2.1.
Update radium to 0.18.0.
Update react-router to 2.4.0.
Update example components to use new code styles.
Add tests and documentation to example components.

### CODE REVIEW

File: examples/components/ScrollList/ScrollList.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# ScrollList
+A list that can scroll through items.
+The list will iterate over the items passed in a create elements based off of the children.
```
Comment (aldeed) [inline examples/components/ScrollList/ScrollList.md]: "...passed in and..."

File: examples/app/components.js [hunk 2]
```diff
@@ -1,21 +1,29 @@
 const components = {};
 
 [
-  'ScrollList',
-  'SearchField',
+  'ScrollList/ScrollList',
+  'SearchField/SearchField',
```
Comment (aldeed) [inline examples/app/components.js]: Why change this if there's an index.js file?

File: examples/components/ScrollList/ScrollList.md (hunk 1 above)
Comment (derekcaneja) [inline examples/components/ScrollList/ScrollList.md]: Fair enough...

File: examples/app/components.js (hunk 2 above)
Comment (derekcaneja) [inline examples/app/components.js]: I kept it this way because it also looks for `.md` and `.tests.jsx`

**Comment by aldeed** [comment]
Looks good other than that one typo. 👍

**Pull request merged**